polaris-ui/src/app/features/cases/api/reauthentication-filter.ts

```diff
@@ -49,7 +60,11 @@ const handleNonAuthCall = (response: Response, window: Window) => {
   return response;
 };
 
-export const reauthenticationFilter = (response: Response, window: Window) =>
-  tryHandleFirstAuthFail(response, window) ||
+export const reauthenticationFilter = (
+  response: Response,
+  window: Window,
+  correlationId?: string
```

Comment:
Have this as `correlationId: string` so the caller is always passing it.  It can still pass `null` or `undefined`